app/main.py

```diff
@@ -1 +1,44 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, appetite=3)
+        self.is_hungry = is_hungry
```

Comment:
Pass `is_hungry` to the `super().__init__` as well, there is no need to redefine it afterward. Apply a similar change to the `Dog` class as well

Reply:
Hi.
![Screenshot 2024-03-28 145701](https://github.com/mate-academy/py-feed-animals/assets/137084131/2f2df681-90ab-44f9-b80e-cdec6a79c235)